packages/utils/src/common/sharding.ts

```diff
@@ -1,4 +1,14 @@
-import type { PubsubTopic, ShardInfo } from "@waku/interfaces";
+import type {
+  PubsubTopic,
+  ShardInfo,
+  SingleTopicShardInfo
+} from "@waku/interfaces";
+
+export const singleTopicShardInfoToPubsubTopic = (
```

Comment:
I think we should be defensive here and check if `shardInfo` has what we need - otherwise throw or fallback